src/Tezos/Forging.cpp

```diff
@@ -23,6 +24,39 @@ Data forgeBool(bool input) {
     return Data{result};
 }
 
+Data forgeInt32(int value, int len) {
+    Data out(len);
+    for (int i = len - 1; i >= 0; i--, value >>= 8) {
+        out[i] = (value & 0xFF);
+    }
+    return out;
+}
```

Comment:
Can we reuse BinaryCoding Encode32LE for this?

Reply:
I've tried it, and it gives a different result :/

Reply:
Hmm, with len=4 Encode32BE should give the same result! With len=1 it is straightforward, other values are not used, as far as I see.

Reply:
should we modify Encode32BE to have a default len of 4, and apply the following logic in order to re-use the same code? or you have smth else in mind?

Reply:
BE